client/extensions/woocommerce/app/order/create.js

```diff
@@ -34,18 +39,37 @@ import { recordTrack } from 'woocommerce/lib/analytics';
 import { sendOrderInvoice } from 'woocommerce/state/sites/orders/send-invoice/actions';
 
 class Order extends Component {
+	possiblyFetchDefaultCurrency( props ) {
```

Comment:
This is personal preference, so feel free to ignore -- but I usually leave the component lifecycle methods at the top of the class (so this would be below `componentWillUnmount`)

Reply:
I like that as well, and it seems a pretty common pattern throughout Calypso.

Reply:
Good idea! Fixed in https://github.com/Automattic/wp-calypso/pull/23568/commits/8a7c403776596e9294f02baf3bf29b7b5c04d140